safe_transaction_service/history/views.py

```diff
@@ -148,6 +148,41 @@ def get(self, request, format=None):
             return Response(self._get_info(ethereum_client))
 
 
+class ERC20IndexingView(GenericAPIView):
+    serializer_class = serializers.ERC20IndexingSerializer
+
+    def get(self, request):
+        """
+        Get current indexing status for ERC20/721 events
+        """
+        current_block_number = EthereumClientProvider().current_block_number
+        min_erc20_block_number = SafeContract.objects.aggregate(
+            min_erc20_block_number=Min("erc20_block_number")
+        )["min_erc20_block_number"]
+        if min_erc20_block_number is None:  # Still nothing indexed
```

Comment:
The default for `erc20_block_number` is `0`. Wouldn't the aggregate never be `None`?

Reply:
You can see the line is covered by the tests 😉 https://coveralls.io/builds/52347914/source?filename=safe_transaction_service%2Fhistory%2Fviews.py#L162